File: poo/Java/Meses/Meses.java
import java.util.Scanner;

public class Meses {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int m = scanner.nextInt();
        switch (m) {
            case 1: System.out.println("JANEIRO"); break;
            case 2: System.out.println("FEVEREIRO"); break;
            case 3: System.out.println("MARCO"); break;
            case 4: System.out.println("ABRIL"); break;
            case 5: System.out.println("MAIO"); break;
            case 6: System.out.println("JUNHO"); break;
            case 7: System.out.println("JULHO"); break;
            case 8: System.out.println("AGOSTO"); break;
            case 9: System.out.println("SETEMBRO"); break;
            case 10: System.out.println("OUTUBRO"); break;
            case 11: System.out.println("NOVEMBRO"); break;
            case 12: System.out.println("DEZEMBRO"); break;
            default: System.out.println("MES INVALIDO"); break;
        }
        scanner.close();
    }
}

File: poo/Cpp/Meses/Meses.cpp
#include <iostream>

int main() {
    int m;
    std::cin >> m;

    switch (m) {
        case 1: std::cout << "JANEIRO" << std::endl; break;
        case 2: std::cout << "FEVEREIRO" << std::endl; break;
        case 3: std::cout << "MARCO" << std::endl; break;
        case 4: std::cout << "ABRIL" << std::endl; break;
        case 5: std::cout << "MAIO" << std::endl; break;
        case 6: std::cout << "JUNHO" << std::endl; break;
        case 7: std::cout << "JULHO" << std::endl; break;
        case 8: std::cout << "AGOSTO" << std::endl; break;
        case 9: std::cout << "SETEMBRO" << std::endl; break;
        case 10: std::cout << "OUTUBRO" << std::endl; break;
        case 11: std::cout << "NOVEMBRO" << std::endl; break;
        case 12: std::cout << "DEZEMBRO" << std::endl; break;
        default: std::cout << "MES INVALIDO" << std::endl; break;
    }

    return 0;
}